operations/iuf/workflows/configuration.md

```diff
@@ -91,14 +91,29 @@ The following highlights some of the areas that require manual configuration cha
 required for initial installation scenarios.
 
 - USS
-    - Configure DVS and LNet with appropriate Slingshot settings
-    - Configure DVS and LNet for use on application nodes
-    - Enable site-specific file system mounts
-    - Set the USS root password in HashiCorp Vault
-- UAN
-    - Enable CAN, LDAP, and set MOTD
-    - Move DVS and LNet settings to USS branch
-    - Set the UAN root password in HashiCorp Vault
+    - Compute Configuration
+        - Configure DVS and LNet with appropriate Slingshot settings
+        - Configure DVS and LNet for use on application nodes
+        - Enable site-specific file system mounts
+        - Set the USS root password in HashiCorp Vault
+    - UAN Configuration
+        - Enable CAN, LDAP, and set MOTD
+        - Move DVS and LNet settings to USS branch
+        - Set the UAN root password in HashiCorp Vault
+        - Enable UAIs on UAN
+    - SLURM Configuration
+        - UAS
```

Comment:
I checked with David and Jeff, and this UAS section can be deleted given the switch from worker nodes to UAIs-on-UANs.  I'm filing a Jira with USS to get our doc section removed as well.  Same comment applies a few line down in the PBS Pro section.